app/src/main/java/net/gsantner/markor/format/txt2tags/Txt2tagsHighlighter.java

```diff
@@ -0,0 +1,201 @@
+/*#######################################################
+ *
+ *   Maintained by Gregor Santner, 2018-
+ *   https://gsantner.net/
+ *
+ *   License of this file: Apache 2.0 (Commercial upon request)
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+#########################################################*/
+package net.gsantner.markor.format.txt2tags;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.text.InputFilter;
+import android.text.Spannable;
+
+import net.gsantner.markor.format.ListHandler;
+import net.gsantner.markor.model.Document;
+import net.gsantner.markor.ui.hleditor.Highlighter;
+import net.gsantner.markor.ui.hleditor.HighlightingEditor;
+
+import java.util.regex.Pattern;
+
+import other.writeily.format.txt2tags.WrTxt2tagsHeaderSpanCreator;
+
+public class Txt2tagsHighlighter extends Highlighter {
```

Comment:
Suggestion: As you said it's mostly the same: extend from ZimHighlighter instead and use inheritance/overrides where required. Guess for example Patterns are 99% same, and we better maintain these only at **one place**.

Reply:
I'm afraid I don't know how to do this inheritance :( I'm not a (java) developper.

It's easier for me to use a diff tool to compare the two in the case there is a modification.

Reply:
extends = Inheritance. extend from zim highliter instead of the generic one